GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -93,11 +93,12 @@ public override void Execute()
                 else
                 {
                     string valueRead = null;
-                    if (!this.localConfig.TryGetConfig(this.Key, out valueRead, out error))
+                    if (!this.localConfig.TryGetConfig(this.Key, out valueRead, out error) ||
+                        string.IsNullOrEmpty(valueRead))

```

Comment:
So we are now saying a empty value is error?

Reply:
Right. Here is a comparison with ```git config```.
```
PS M:\Work\GVFS\VFSForGit_Ameen> git config un.defined.key "  "
PS M:\Work\GVFS\VFSForGit_Ameen> git config un.defined.key

PS M:\Work\GVFS\VFSForGit_Ameen> echo $?
True
PS M:\Work\GVFS\VFSForGit_Ameen> gvfs config un.defined.key "  "
PS M:\Work\GVFS\VFSForGit_Ameen> gvfs config un.defined.key

PS M:\Work\GVFS\VFSForGit_Ameen> echo $?
True
PS M:\Work\GVFS\VFSForGit_Ameen>
```

Reply:
Looks like we should be able to write some tests around this right?

Reply:
Added Functional tests.